packages/opensrp-plans/src/components/PlanInfo/index.tsx

```diff
@@ -6,7 +6,7 @@ import { pageTitleBuilder } from '../../containers/ListView/utils';
 import { EDIT, END_DATE, MISSIONS, START_DATE } from '../../lang';
 import React from 'react';
 import { Link } from 'react-router-dom';
-import './index.css';
+import '../../index.css';
```

Comment:
lets remove the internal css import declarations, my idea is to have them optionally imported and used where the packages is being consumed